src/utils/folderutils.cpp

```diff
@@ -117,6 +118,23 @@ QDir FolderUtils::getAppPartsSubFolder(QString search) {
 
 QDir FolderUtils::getAppPartsSubFolder2(QString search) {
 	if (m_partsPath.isEmpty()) {
+#ifdef PKGDATADIR
+		QStringList candidates;
+		candidates.append(QDir::homePath() + "/.local/share/fritzing");
```

Comment:
Please don't hardcode `~/.local/share`. It's a fallback value for cases when `$XDG_DATA_HOME` env var is not set. See the [spec](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html).